compiler/src/iree/compiler/InputConversion/MHLO/ConvertMHLOToLinalgExt.cpp

```diff
@@ -228,6 +233,37 @@ struct ScatterOpConversion : public OpConversionPattern<mhlo::ScatterOp> {
     return true;
   }
 
+  static Value insertImplicitTrailingDimIfNeeded(
+      mhlo::ScatterOp op, ConversionPatternRewriter &rewriter,
+      OpAdaptor adaptor) {
+    auto scatterIndices = adaptor.scatter_indices();
+    auto scatterDims = adaptor.scatter_dimension_numbers();
+    auto indicesType = scatterIndices.getType().cast<ShapedType>();
+    auto indicesShape = indicesType.getShape();
+    auto indicesRank = indicesType.getRank();
+    auto indexVectorDim = scatterDims.getIndexVectorDim();
+
+    if (indexVectorDim == indicesRank) {
```

Comment:
Is only 1 trailing dim implicit dim allowed?

Reply:
As far as I understand, yes. (Plus, if it's implicit - how would we know to differentiate between one or more? :)

Reply:
👍